src/EventHandler/EventHandlerBase.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\EventHandler\EventHandlerBase.
+ */
+
+namespace Drupal\rules\EventHandler;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\rules\Core\RulesConfigurableEventHandlerInterface;
+use Drupal\rules\Core\RulesDefaultEventHandler;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Base class for event handler.
+ */
+abstract class EventHandlerBase extends RulesDefaultEventHandler implements RulesConfigurableEventHandlerInterface {
```

Comment:
Should be ConfigureableEventHandlerBase  - as that what it implements. same for comment.